includes/page/Article.php

```diff
@@ -488,6 +488,13 @@ public function view() {
 		];
 		# Allow extensions to vary parser options used for article rendering
 		Hooks::runner()->onArticleParserOptions( $this, $parserOptions );
+
+		/**
+		 * Fandom change - start (@author miwaniszczuk) - UGC-4161
+		 * Make EditSectionLink Hook run for semi-protected pages
+		 */
```

Comment:
**suggestion (non-blocking)**: IMO this information is in commit message (the ticket number, who made change and why). No need to duplicate this in code.